### DISCUSSION TITLE
自定义字典，中文识别visionlan只输出数字

### DISCUSSION THREAD

**Discussion by peidongha** [PaddlePaddle/PaddleOCR#12501] [Q&A]
版本是PPOCR2.7
自定义了字典，字典内容是700多个中文字符和0-9，训练和推理都用了自定义字典，train和validation的准确率都没有什么问题，测试时也没有报错。
但是运行一下指令：
python tools/infer/predict_system.py --image_dir="./train_data/icdar2015/text_localization/test_data/" --det_model_dir="./inference_model/ch_PP-OCRv3_det_infer/Student2/"  --rec_model_dir="./inference_model/ch_PP-OCRv3_rec_distillation_inference_adc/Teacher/
只能输出数字，无法输出中文
按照#7960的方法修改后，还是一样的问题。
想问一下有什么解决方法或者可以尝试的方向吗？
谢谢~

**Comment by peidongha** [comment]
之所以自定义字典，是因为700多个中字符没有包括在默认的中文字典中，所以重新定义了字典，使用的ch_PP-OCRv3_rec_train预训练模型做的fine-tune

**Comment by peidongha** [comment]
![412354490_1](https://github.com/PaddlePaddle/PaddleOCR/assets/37918885/be2f79e4-59c5-40c5-b1f6-ccbb902814f0)

**Comment by hezhenname** [comment]
你自定义字典，参数不用加吗

**Comment by peidongha** [comment]
> 你自定义字典，参数不用加吗

我用的是ch_PP-OCRv3_rec_distillation.yml配置文件，只修改了字典加载路径，数据加载路径，还有显卡的配置，其他没有修改过

**Comment by hezhenname** [comment]
> 数据加载路径，还有显卡的配置，其

我是说你跑预测的时候，不用加字典路径吗

**Comment by peidongha** [comment]
> > 数据加载路径，还有显卡的配置，其
> 
> 我是说你跑预测的时候，不用加字典路径吗
<img width="395" alt="a8f4125c1d27485319e98d559b1e395" src="https://github.com/PaddlePaddle/PaddleOCR/assets/37918885/2b8aedc3-027a-4bb3-99d6-ca9391ed9774">

跑预测也添加字典路径了，和训练时的字典一致

**Comment by xuxiansheng2018** [comment]
我也遇到相似的问题，我只在字典里面新增了一个字，但是我识别的时候，比如“20号”识别出来“20”，不知道如何解决，希望有大佬解答

**Comment by anbo724** [comment]
训练的时候使用预训练模型吗？还是完全冲头训练？
